CMakeLists.txt

```diff
@@ -3,6 +3,9 @@ cmake_minimum_required (VERSION 3.12)
 project(KokkosRemote
   LANGUAGES CXX
   VERSION 0.1.0)
+  
+message(STATUS "Setting policy CMP0074 to use <Package>_ROOT variables")
+cmake_policy(SET CMP0074 NEW)
```

Comment:
Why would that be needed?

Reply:
your minimum cmake requirement for your project is 3.12 so you don't need it

Reply:
Thanks